pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1687,70 +1690,49 @@
     ::  +sign-to-channel-event: strip the vase from a sign:agent:gall
     ::
     ++  sign-to-channel-event
-      |=  =sign:agent:gall
-      ^-  channel-event
-      ?.  ?=(%fact -.sign)  sign
-      [%fact [p q.q]:cage.sign]
+      |=  [=sign:agent:gall =channel request-id=@ud]
+      ^-  (unit channel-event)
+      ?.  ?=(%fact -.sign)  `sign
+      ?~  desk=(app-to-desk channel request-id)  ~
```

Comment:
As discussed with @Fang-, we might be able to just assert here. I did notice that crashing here is a road straight to brick city with `behn: timer failed; queue blocked` so I decided to keep this unitized.